code/search/indexes/SearchIndex.php

```diff
@@ -540,9 +565,10 @@ protected function _getFieldValue($object, $field)
      * @param Exception $e
      * @throws Exception
      */
-    public static function warn($e) {
+    public static function warn($e)
+    {
         // Noisy errors during testing
-        if(class_exists('SapphireTest', false) && SapphireTest::is_running_test()) {
+        if (class_exists('SapphireTest', false) && SapphireTest::is_running_test()) {
```

Comment:
The SapphireTest reference here isn't going to work as it's not namespaced, and the `is_running_test` API is removed. That said, it's not throwing exceptions in the test suite at the moment, so perhaps it can just be removed.